### ISSUE TITLE
investigate issue with reset settings dialogue

### ISSUE THREAD

**Issue by reginacas** [reginacas/sense-ui#86]
> I tried to reset settings. It only announces the title The extension SenseUI says  dialog. However, nvda does not finish reading the message that what will happen. I have to use object navigation to read it. NVDA should able to read the complete message without object navigation. I resetted settings, it resetted most settings but did not change the keyboard shortcut. 

 _Reported by @fahimnet2014 in [#71](https://github.com/reginacas/sense-ui/issues/71#issuecomment-3738116547)_

This was also flagged by @ammarawaisi , and his report and suggestion is the following:

> Accessibility: When activated, a native confirm dialog appears. The description text in this dialog was not focusable via Tab/Shift+Tab; I had to use NVDA Object Navigation to read it. I suggest creating a custom, fully accessible modal.
